elooplib/client.cpp

```diff
@@ -0,0 +1,65 @@
+#include "client.hpp"
+
+Client::Client(size_t d) : Node(d){};
+Client::Client(Client&& rhs) : Node(std::move(rhs)),
+    request_field(std::move(rhs.request_field)) {};
```

Comment:
You are moving rhs.request_field of already moved rhs, this looks very suspicious, please fix